omod-2.4/src/main/java/org/openmrs/module/webservices/rest/web/v1_0/resource/openmrs1_8/PatientResource1_8.java

```diff
@@ -195,6 +195,18 @@ public DelegatingResourceDescription getCreatableProperties() {
 		DelegatingResourceDescription description = new DelegatingResourceDescription();
 		description.addRequiredProperty("person");
 		description.addRequiredProperty("identifiers");
+
+		if (Context.hasPrivilege(RestConstants.PRIV_SET_AUDIT_DATA)) {
+			description.addProperty("creator");
+			description.addProperty("dateCreated");
+			description.addProperty("changedBy");
+			description.addProperty("dateChanged");
+			description.addProperty("personDateCreated");
+			description.addProperty("personCreator");
+			description.addProperty("personDateChanged");
+			description.addProperty("personChangedBy");
+		}
+
```

Comment:
Anything stopping you to apply it generally to all resources in DelegatingCrudResource and DelegatingSubResource for objects implementing Auditable/Retireable/Voidable?